packages/ember-glimmer/lib/helpers/hash.js

```diff
@@ -0,0 +1,38 @@
+import { helper } from '../helper';
+/**
+@module ember
+@submodule ember-templates
+*/
+
+/**
+   Use the `{{hash}}` helper to create a hash to pass as an option to your
+   components. This is specially useful for contextual components where you can
+   just yield a hash:
+
+   ```handlebars
+   {{yield (hash
+      name='Sarah'
+      title=office
+   )}}
+   ```
```

Comment:
I added these and they seem to Just Work™. I wrote tests for internal, external, and mixed hash yields.